generated-platform-project/quarkus/bom/pom.xml

```diff
@@ -11482,6 +11482,11 @@
         <artifactId>jboss-threads</artifactId>
         <version>3.6.1.Final</version>
       </dependency>
+      <dependency>
+        <groupId>org.jboss</groupId>
+        <artifactId>jandex</artifactId>
+        <version>3.2.0</version>
+      </dependency>
```

Comment:
@ppalaga do you know where this is coming from?

Reply:
Ok, looks like if you rebase it will go away

Reply:
Ok, it looks like `org.jboss:jandex:3.2.0` was published on 2024-07-29, while `io.smallrye:jandex:3.2.0` was published on 2024-05-16.